docs/data_usage_guides/enum_parameters/price_flag_parameters.yaml

```diff
@@ -0,0 +1,3 @@
+- price flag: actual
+- price flag: aggregate
+- price flag: actual,aggregate
```

Comment:
actual data is collected originally with monthly frequency; aggregated data is collected with higher frequency (daily or weekly) averaged through a weighted mean to monthly; imputed data is an estimation of missing values in the time series, obtained through the R Amelia-II package; forecasted data is a six months prediction obtained through different methods, which are reported in the API.

(We only collect actual and aggregate)